src/lwc/geFormField/geFormField.js

```diff
@@ -461,45 +439,22 @@ export default class GeFormField extends LightningElement {
         this.dispatchEvent(formFieldChangeEvent);
     }
 
-    recordTypeNameFor(recordTypeId) {
```

Comment:
May need to update the remaining call site for `recordTypeNameFor` around line 435 to reference `this.objectDescribeUtil`